README.md

```diff
@@ -25,7 +25,8 @@ All other fields must be provided.
 "continueUrl" : "/test",
 "optServiceName" : "Service Name",
 "deskProServiceId" : "abc",
-"signOutUrl : "/sign-out"
+"signOutUrl" : "/sign-out",
+"applicantVerification" = "onlyNino"
```

Comment:
This isn't valid JSON, there also needs to be an explanation of what this field does in the readme. ApplicantVerification isn't very clear as to what this is for, I would name it something like this.
```suggestion
"enableSautrCheck": true 
The enableSautrCheck field allows the calling service to decide whether to ask the user to provide an SAUTR that will be verified.
```
Using a string here means we open the app up to issues when something we haven't thought of is sent. There's no type safety there. As much as I don't like it having 3 states because 2 are essentially the same, an Option[Boolean] is probably the best solution.